scripts/github-actions/tests-e2e-direct.sh

```diff
@@ -24,24 +24,23 @@ cd ${REPO_ROOT}/
 echo "Downloading envtest assets..."
 export KUBEBUILDER_ASSETS=$(go run sigs.k8s.io/controller-runtime/tools/setup-envtest@latest use -p path)
 
+#export KCC_USE_DIRECT_RECONCILERS=LoggingLogMetric
+
 echo "Running e2e tests samples for LoggingLogMetric direct reconciliation..."
 
-KCC_USE_DIRECT_RECONCILERS=LoggingLogMetric \
```

Comment:
ah! you bit me. I fix this in my WIP PR.

Reply:
Happy to merge yours first if you prefer!